src/DataGrid.tsx

```diff
@@ -208,6 +209,7 @@ function DataGrid<R, SR>({
   // Toggles and modes
   enableFilterRow = false,
   cellNavigationMode = 'NONE',
+  enableVirtualization = true,
```

Comment:
Not sure on what the prop name. `disableVirtualization` makes more sense but I kept this to be consistent with `enableHeaderRow` . We can also consider forwarding this prop to the formatters to users can use it during printing.

Reply:
Prop name is good to me. I prefer avoiding "negative" prop/variable names as they make reading logic harder.

>We can also consider forwarding this prop to the formatters to users can use it during printing.

What use case do you have in mind?

Reply:
If for example there is a cell that shows a complex data like an icon with a tooltip. It may not work out of the box and it would helpful to provide an alternate formatter for printing. We can revisit if needed

Reply:
Sounds like it should be a separate thing, and it could be implemented by the user via a provider.